tests/foreman/api/test_contentviewfilter.py

```diff
@@ -57,3 +81,265 @@ def test_get_with_bad_args(self):
             response.status_code,
             (httplib.BAD_REQUEST, httplib.UNPROCESSABLE_ENTITY)
         )
+
+    @data(
+        gen_string('alphanumeric', randint(1, 255)),
+        gen_string('alpha', randint(1, 255)),
+        gen_string('cjk', randint(1, 85)),
+        gen_string('latin1', randint(1, 255)),
+        gen_string('numeric', randint(1, 255)),
+        gen_string('utf8', randint(1, 85)),
+        gen_string('html', randint(1, 85)),
```

Comment:
I see this same bit of data being passed in to every single function. You could clean this up by using a private function. I think this works:

```python
def _data():
    return (
        gen_string('alphanumeric', randint(1, 255)),
        …
    )

@data(*_data())
def test_create_erratum_cvf_with_different_names(self, name):
    …
```

Even better, see whether there's some even more general function that can be used, much as the CLI tests do.